brands/shop/clothes.json

```diff
@@ -2406,6 +2406,15 @@
       "name": "Polo Ralph Lauren",
       "shop": "clothes"
     }
+  },
+   "shop/fashion_accessories|Prada": {
```

Comment:
```suggestion
   "shop/clothes|Prada": {
```

Reply:
I think when I added this one originally it was because the stores also sell fragrances and bags and other things (maybe people buy those more than the clothes, I'm not sure).